AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/CallActivity.java

```diff
@@ -70,6 +70,11 @@
     private boolean callHangUpOverlaid;
     private Button callHangupConfirmButton;
     private Runnable initialVideoToggleRequest;
+    private ConstraintLayout listMenuLayer;
```

Comment:
may be deviceListMenuLayer?

Reply:
Same as previous reply, this layer will be re-used for participants' list menu later, so it is better not to name it only for device.